x-pack/plugin/autoscaling/src/internalClusterTest/java/org/elasticsearch/xpack/autoscaling/existence/FrozenExistenceDeciderIT.java

```diff
@@ -138,6 +139,9 @@ public void testZeroToOne() throws Exception {
         // we've seen a case where bootstrapping a node took just over 60 seconds in the test environment, so using an (excessive) 90
         // seconds max wait time to avoid flakiness
         assertBusy(() -> {
+            // cause a bit of cluster activity using an empty reroute call in case the `wait-for-index-colour` ILM step missed the
+            // notification that partial-index is now GREEN.
+            client().admin().cluster().reroute(new ClusterRerouteRequest());
```

Comment:
Do you want to add the `.get()` on this to block until the reroute finishes? I don't think it'd actually cause this to fail, since even if the reroute is async it would eventually either be retried or the asserts will succeed, but it may be nicer to wait.